src/test/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotationsIntegrationTest.java

```diff
@@ -52,7 +63,8 @@ public void testFilterUniform(final String vcfName,
         final List<String> args = Arrays.asList(
                 "-V", TEST_DATA_DIR.resolve(vcfName).toString(),
                 "-O", tmpOut.toString(),
-                "--ref-version", ref.name()
+                "--ref-version", ref.name(),
```

Comment:
Have you see the `ArgumentsBuilder` class?  It lets you do some arguments setup a little more cleanly.  Might be worth looking into for next time.